lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -1613,6 +1613,101 @@ export class BrowserCacheManager extends CacheManager {
         this.setInteractionInProgress(false);
     }
 
+    /**
+     * Create request retry key to cache retry status
+     * @param correlationId
+     */
+    generateRequestRetriedKey(correlationId: string): string {
+        return `${Constants.CACHE_PREFIX}.${TemporaryCacheKeys.REQUEST_RETRY}.${correlationId}`;
+    }
+
+    /**
+     * Gets the request retry value from the cache
+     * @param correlationId
+     */
+    getRequestRetried(correlationId: string): number | null {
+        const requestRetriedKey = this.generateRequestRetriedKey(correlationId);
+        const cachedRetryNumber = this.getTemporaryCache(requestRetriedKey);
+        if (!cachedRetryNumber) {
+            return null;
+        }
+        return JSON.parse(cachedRetryNumber);
+    }
+
+    /**
+     * Sets the request retry value to "retried" in the cache
+     * @param correlationId
+     */
+    setRequestRetried(correlationId: string): void {
+        this.logger.trace("BrowserCacheManager.setRequestRetried called");
+        const requestRetriedKey = this.generateRequestRetriedKey(correlationId);
+        this.setTemporaryCache(requestRetriedKey, JSON.stringify(1), false);
```

Comment:
We can use the static 1 here to save an extra operation

```suggestion
        this.setTemporaryCache(requestRetriedKey, "1", false);
```